index.bs

```diff
@@ -5057,24 +4996,13 @@ The privacy, security and operational characteristics of [=attestation=] depend
     operating environment, and so on.
 
 The [=attestation type=] and [=attestation statement format=] is chosen by the [=authenticator=];
-[=[RPS]=] can only signal their preferences by setting the {{PublicKeyCredentialCreationOptions/attestation}} and {{PublicKeyCredentialCreationOptions/attestationFormats}} parameters, or those with the same names in {{PublicKeyCredentialRequestOptions}}.
+[=[RPS]=] can only signal their preferences by setting the {{PublicKeyCredentialCreationOptions/attestation}} and {{PublicKeyCredentialCreationOptions/attestationFormats}} parameters.
```

Comment:
Above this, [near the beginning of the "Attestation" section](https://github.com/w3c/webauthn/pull/1997/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985L5013), remains some language about "attestation is returned when a credential is created, and optionally exercised". See the diff that introduced it: https://github.com/w3c/webauthn/pull/1663/commits/20dd35c00a4caf07b1514c5495afaaf1482e04ae#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985L4308-R4358

Reply:
Similarly, I think we can revert the change to the `<figcaption>` in that section too: https://github.com/w3c/webauthn/pull/1663/commits/20dd35c00a4caf07b1514c5495afaaf1482e04ae#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985L4319-R4368

Reply:
Thanks. Done.